app/controllers/pets_controller.rb

```diff
@@ -29,8 +29,12 @@ def edit
   def update
     pet = Pet.find(params[:id])
     pet.update(pet_params)
-    pet.save
-    redirect_to "/pets/#{pet.id}"
+    if pet.save
+      redirect_to "/pets/#{pet.id}"
```

Comment:
These first few lines can be condensed into ` if Pet.update(params[:id], pet_params).save`

Reply:
Yep. I did this for the other update methods as well